config/prow/cluster/monitoring/alertmanager-prow_secret.yaml

```diff
@@ -19,11 +19,17 @@ stringData:
       repeat_interval: 4h
       receiver: 'slack-warnings'
       routes:
+      - receiver: 'cluster-api-aws-alerts'
+        group_interval: 5m
+        repeat_interval: 2h
+        matchers:
+          - boskos_type = aws-account
       - receiver: 'slack-alerts'
         group_interval: 5m
         repeat_interval: 2h
-        match_re:
-          severity: 'critical|high'
+        matchers:
+          - severity =~ "critical|high"
```

Comment:
This didn't need to change, but it looks equivalent according to the docs.

Reply:
I did it for completeness since `match_re` is marked as deprecated.